lib/logstash/outputs/datadog_metrics.rb

```diff
@@ -64,7 +64,7 @@ def register
     )
   end # def register
 
-  public
+  # public
```

Comment:
I understand why do you add this comments? But I would recommend to remove them before merging, otherwise we'll end up with lots of comments like this, that end up providing not much of intel to the reader.